### DISCUSSION TITLE
any way to get multiple cameras to work?

### DISCUSSION THREAD

**Discussion by cruzer619** [nlef/moonraker-telegram-bot#291] [General]
I have a nozzle cam and pi cam.. would like to see both pictures comeup in status.  Thanks.

**Comment by cruzer619** [comment]
i'm surpised we can't still do this yet.  In the octoprint version you can have multiple cameras.

**Comment by nlef** [comment]
I was asked for this functionality for a long time. At the moment, neither I nor the bot users testing it have ready-made stands for working with several cameras, so the development and testing of this functionality is not a priority
I am always happy to receive pull requests with expanded functionality

As soon as this functionality is implemented, it will be reflected in the bot's release notes

**Reply by miniraser** [reply]
hello everyone! I'm also interested in the possibility of using multiple cameras. if necessary, I can provide the data and my printer for testing this development. PS I really don't understand anything about development)
